lib/DockerWrapper.js

```diff
@@ -61,6 +61,24 @@ const getCurrentServiceConfig = (service) => {
   });
 };
 
+const generateServiceEnvVars = async (app) => {
+    const hasDatabase = await app.getDatabase();
```

Comment:
```suggestion
    const hasDatabase = app.getDatabase() !== null;
```

Reply:
My suggestion here is just to save a database query.

Reply:
I'll try this out later! I was initially thinking that `.getDatabase()` returned a promise regardless of whether a database existed.

Reply:
Just tested it out and `app.getDatabase()` always returns a Promise, so `app.getDatabase() !== null` will always evaluate to `true` regardless of whether there is actually a database.

Reply:
Though I suppose we could just do `app.databaseId !== null` (why Sequelize doesn't have a method like this that immediately returns `true`/`false` I'm not sure)